example/macos/example/App.tsx

```diff
@@ -0,0 +1,88 @@
+/**
```

Comment:
curious why do we need a separate App.tsx and not use the main one?

Reply:
At first it was my understanding that we weren't able to combine the macOS example app with the others because of which version of RN we had to pull to get things working. After further investigation (and a bit more experience with these projects), I think it might be possible to combine these after all. I'll look into this, and if it works out, I'll update this PR.

Reply:
Updated.